### PR TITLE
Added kubernetes deployment

### PR DESCRIPTION
**Pull request by vitalykarpenko**
Added logic for health and readiness checks

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -90,3 +90,20 @@ vendor:
 .PHONY: vendor-update
 vendor-update:
 	$(BUILDER) dep ensure
+
+.PHONY: image
+image:
+	docker build -f docker/Dockerfile.server -t infobloxopen/server:v1.0 .
+	docker build -f docker/Dockerfile.gateway -t infobloxopen/gateway:v1.0 .
```
Comment (Evgeniy-L) [inline Makefile]: We do not have infobloxopen hub, I expect it should be infoblox. Also I would prefer image name that is more informative, for example infoblox/contacts-server:v1.0 & infoblox/contacts-gateway:v1.0

File: README.md [hunk 2]
```diff
@@ -10,67 +10,110 @@ These instructions will get you a copy of the project up and running on your loc
 
 ### Prerequisites
 
-#### Local setup:
+#### Local setup
+
 Need `mysql` or `mariadb` installed.
```
Comment (Evgeniy-L) [inline README.md]: Why `mysql` or `mariadb`?

File: README.md [hunk 3]
```diff
@@ -10,67 +10,110 @@ These instructions will get you a copy of the project up and running on your loc
...
+``` sh
+    systemctl start mariadb
 ```
 
 Create `atlas-contacts-app` database:
 
-```
-    $ mysql -u root
+``` sh
+    mysql -u root
```
Comment (Evgeniy-L) [inline README.md]: I thought we moved to postgress. Is this the case?

File: kube/kube.yaml [hunk 4]
```diff
@@ -0,0 +1,178 @@
...
+        args: 
+        - "-health"
+        - "0.0.0.0:8088"
+        - "-shealth"
+        - "0.0.0.0:8089"
+        image: infoblox/contacts-gateway:v1.0
+        imagePullPolicy: IfNotPresent
+        ports:
+        - containerPort: 8080
+        # livenessProbe:
```
Comment (Evgeniy-L) [inline kube/kube.yaml]: why this is commented out?

File: README.md (hunk 2 above)
Comment (vitalykarpenko) [reply README.md]: It's for local setup. Forgot to remove it. We're having kubernetes setup since now.

**Pull request merged**